packages/okta-auth-js/lib/browser/browser.js

```diff
@@ -181,7 +181,7 @@ function OktaAuthBuilder(args) {
 var proto = OktaAuthBuilder.prototype;
 proto._onTokenManagerError = function(error) {
   var code = error.errorCode;
```

Comment:
I wasn't the author of the below, so if Aaron was I'll let him say if there was a particular reason, but otherwise this lgtm

Reply:
I think it's a good change. As I've mentioned, I believe we should retire `onSessionExpired` in favor of just listening to the "error" event for token renew. There are other reasons token renew can fail, such as 3rd party cookies being blocked. When we move to active autoRenew we can decide how/if we want to handle this. The passive approach should mean that token validity will be evaluated at the time of a route change or other specific user action, and the result can be retrieved within a promise chain. Starting async logic from an event handler is not a good pattern

Reply:
But logically, there is no difference between accessToken and idToken. As I recall I wrote it this way because I assumed accessToken renew would occur before/more frequently than idToken and if they occurred at the same time I did not want two events to be fired (because that would start two login flows).

Reply:
If we are planning on retiring this event/callback in the next major version, maybe we should leave it as-is to avoid chance that it causes unexpected results (such as the double login flow)